### PR TITLE
chore: Add the fastify to the testing suite.

### PR DESCRIPTION
**Pull request by lholmquist**
using this instead of #382

### CODE REVIEW

**Comment by lholmquist** [comment]
[test]

**Comment by lholmquist** [comment]
looks like this is failing becuase it can't find the functions?

```
-----------------------------------------------
Running test test_client_cloudevents (starting at 2023-05-11 14:15:42+00:00) ... 
-----------------------------------------------
test/test-lib.sh: line 1300: test_client_cloudevents: command not found
Test for image 'quay.io/fedora/nodejs-18:18' FAILED (exit code: 127)
-----------------------------------------------
Running test test_client_fastify (starting at 2023-05-11 14:15:42+00:00) ... 
-----------------------------------------------
test/test-lib.sh: line 1300: test_client_fastify: command not found
Test for image 'quay.io/fedora/nodejs-18:18' FAILED (exit code: 127)
```

@zmiklank i only did a rebase on the upstream,  not sure why this started happening

**Comment by zmiklank** [comment]
[test]

**Comment by zmiklank** [comment]
This is strange. I also tried to run the tests locally. Result was the same -- apparently caused by the last commit.
I need to look into this closer.

**Comment by phracek** [comment]
It looks like something is really strange:

```bash
npm notice 
test/test-lib-nodejs.sh: line 578: syntax error: unexpected end of file
hint: Using 'master' as the name for the initial branch. This default branch name
h
```

**Comment by phracek** [comment]
@lholmquist Can you please add also an updated common submodule? Only for sure.

File: test/test-lib-nodejs.sh [hunk 1]
```diff
@@ -417,6 +417,10 @@ function test_client_faas() {
 function test_client_cloudevents() {
   echo "Running CloudEvents client test"
   test_running_client_js cloudevents
+
```
Comment (zmiklank) [inline test/test-lib-nodejs.sh:420]: Missing bracket.
```suggestion
}
```

Comment (lholmquist) [reply test/test-lib-nodejs.sh]: 🤦

**Comment by lholmquist** [comment]
[test]

**Review by zmiklank — APPROVED** [review]

**Pull request merged**